client/consensus/slots/src/lib.rs

```diff
@@ -194,7 +195,11 @@ pub trait SimpleSlotWorker<B: BlockT> {
 
 		let authorities_len = self.authorities_len(&epoch_data);
 
-		if !self.force_authoring() && self.sync_oracle().is_offline() && authorities_len > 1 {
+		if authorities_len.is_none() ||
+			(!self.force_authoring() &&
+			 self.sync_oracle().is_offline() &&
+			 authorities_len.map(|a| a > 1).unwrap_or(true))
```

Comment:
```suggestion
			 authorities_len.map(|a| a > 1).unwrap_or(false))
```

Reply:
I think we can remove initial `authorities_len.is_none()` check if we do this instead.